src/diagonal.works/b6/api/functions/collections_test.go

```diff
@@ -239,14 +239,14 @@ func TestFlattern(t *testing.T) {
 		Values: []interface{}{&c1, &c2},
```

Comment:
L 228 / test name also

Reply:
Thanks! I found a couple of others with `find . -name \*.py -or -name \*.go | xargs grep -i flattern`, which I should have done in the first place, so fixed those too.